starter-code/js/index.js

```diff
@@ -1,11 +1,52 @@
-var $cart = document.querySelector('#cart tbody');
-var $calc = document.getElementById('calc');
+var $cart = document.querySelector("#cart tbody");
+var $calc = document.getElementById("calc");
+var $create = document.getElementById("create");
 
-function updateSubtot($product) {
+var $delete = document
+  .querySelectorAll(".rm > button")
+  .forEach(function(button) {
+    button.onclick = deleteProduct;
+  });
+
+function updateSubtot(element) {
   // Iteration 1.1
+  let subTotal;
+  let cost = element.querySelector(".pu > span").innerText;
+  let quantity = element.querySelector(".qty > label > input").value;
+  let subtotDiv = element.querySelector(" td.subtot > span");
+
+  subtotDiv.innerHTML = cost * quantity;
+  return cost * quantity;
 }
 
 function calcAll() {
+  let totalValue = 0;
   // Iteration 1.2
+  document.querySelectorAll("#cart > tbody > tr").forEach(element => {
+    totalValue += updateSubtot(element);
+  });
+
+  document.querySelector("body > h2 > span").innerHTML = totalValue;
+}
+
+function deleteProduct(event) {
+  let currentEventTarget = event.currentTarget; //the delete button you clıcked on
+
+  if (currentEventTarget.parentNode) {
+    currentEventTarget.parentNode.parentNode.parentNode.removeChild(
```

Comment:
This seems a bit messy as it can be difficult for someone to understand what you are actually removing. 
I think it would be cleaner to write something like:
document.querySelector("#cart > tbody").removeChild(currentEventTarget.parentNode.parentNode)
However it is still unclear what is actually being removed here.

Below is another way of solving the delete button challenge that may be a bit more clear to any person in the future to what is going on in the code. If this way or your way is the best is something that can always be discussed:)

function addDeleteOnClick(element) {
  element.querySelector(".rm > button").onclick = function(){
    document.querySelector("#cart > tbody").removeChild(element)
  }
}

document.querySelectorAll("#cart > tbody > tr").forEach((ele) => {
  addDeleteOnClick(ele)
})